src/main/java/frc/team166/robot/subsystems/Manipulator.java

```diff
@@ -188,4 +213,36 @@ protected void interrupted() {
             }
         };
     }
+
+    public Command DeployManipulator() {
+        return new SubsystemCommand("Deploy Manipulator", this) {
+
+            @Override
+            protected void initialize() {
+                setSetpoint(
+                        Preferences.getInstance().getDouble(RobotMap.Preferences.MANIPULATOR_HORIZONTAL_INPUT, 2.5));
+            }
+
+            @Override
+            protected void execute() {
+
+            }
+
+            @Override
+            protected boolean isFinished() {
+
+                return isTimedOut();
```

Comment:
I think you need to set this somewhere first? What is the timeout value??

Reply:
No, it is just a function. It isn't a variable or anything that needs to be declared somewhere else.

Reply:
Nevermind I'm wrong, just fixed it.